src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -1117,6 +1117,8 @@ external gteq   ::= collapse GTEQ GT EQ       { name = ">=" }
 external ltlteq ::= collapse LTLTEQ LT LT EQ  { name = "<<=" }
 external ltlt   ::= collapse LTLT LT LT       { name = "<<" }
 external lteq   ::= collapse LTEQ LT EQ       { name = "<=" }
+external oror   ::= collapse OROR OR OR       { name = "||" }
+external or     ::= onlySingle OR             { name = "|" }
```

Comment:
Are you sure that we need this rule? Seems like `bool_or_bin_expr`  has a higher precedence then `bit_or_bin_expr` and so we don't need to check that `|` is single. I had not tested this and I always  forget which is high and which is low precedence, so I might be mistaking here :)